src/main/webapp/app/exercises/shared/result/result.component.ts

```diff
@@ -267,12 +267,12 @@ export class ResultComponent implements OnInit, OnChanges {
     private determineShowMissingAutomaticFeedbackInformation(componentInstance: FeedbackComponent) {
         if (!this.latestIndividualDueDate) {
             this.exerciseService.getLatestDueDate(this.exercise!.id!).subscribe((latestIndividualDueDate?: dayjs.Dayjs) => {
-                this.latestIndividualDueDate = latestIndividualDueDate;
-                this.initializeMissingAutomaticFeedbackAndLatestIndividualDueDate(componentInstance);
+                // If the server returns no value, a not-reachable date in the future is assumed
+                this.latestIndividualDueDate = latestIndividualDueDate ?? dayjs().add(999, 'years');
```

Comment:
If the exercises did not have a due date or no participation had an individual due date, the `exerciseService` would return `null`, leading to repeated calls to that rest call

Reply:
Why are there repeating calls? I personally would try to fix whatever is causing the repeated calls and allow undefined here. For all the date fields undefined is a possible value that just indicates no deadline, so the logic here should also handle this case.

Reply:
it's right above this code snippet. There is a `if(!this.latestIndividualDueDate)`